### PR TITLE
new blog post

### PR DESCRIPTION
**Pull request by kmugrage**
Trying this again, the first try didn't get committed correctly for some reason.

Open to comments / proofreading if people have cycles.

### CODE REVIEW

File: _posts/2015-06-23-hardly-anyone-knows-cd.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+author: Ken Mugrage
+---
+
+####Those of us who work in or around teams doing continuous delivery often think it’s a mainstream thing. This couldn’t be further from the truth.
+
+I work for ThoughtWorks, a company that implements processes and technologies we think are good long before most. We built the first CI server with Cruise Control, and Go was the first purpose built Continuous Delivery server. I go to a lot of conferences and events, read a lot of blogs, talk to a lot of peers, work with a lot of partners, etc. I thought most people involved with the creation of software had a pretty good idea what CD is.
+
+I was wrong.
+
+I just got back from a pretty big software conference that was a bit off my normal track. They had a few DevOps sessions this year, but historically this particular conference has been more about agile methodologies. As a sponsor of the event, I spent most of my time standing in the booth talking to people.
```
Comment (ketan) [inline _posts/2015-06-23-hardly-anyone-knows-cd.markdown]: Rephrase as - "As one of the sponsors, I spent a lot of time at the booth talking to people"

File: _posts/2015-06-23-hardly-anyone-knows-cd.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Hardly Anyone Knows Continuous Delivery
```
Comment (ketan) [inline _posts/2015-06-23-hardly-anyone-knows-cd.markdown]: Rephrase as "Not everyone knows what CD is". I'm just trying to mellow it down a bit :)

File: _posts/2015-06-23-hardly-anyone-knows-cd.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Take a friend who’s CD impaired to a [DevOps Meetup](http://devops.meetup.com/). As I’m writing this there are groups in 404 cities worldwide at that link alone. Trying to get your meetup going and struggling for content and/or speakers? Tell me, I know a few people and might be able to help.
+
+Stop assuming everyone knows what we’re talking about when we talk about CD. Many of them are smiling and nodding the same way I do when my mother talks about her flowers.
+
+Feel free to comment with your own resource, this isn't even close to a definite list.
+
+####One last thing...
+
+Stop telling people that the phrases DevOps and Continuos Delivery are overused. They aren’t. Hardly anyone knows what Continuous Delivery is.
```
Comment (ketan) [inline _posts/2015-06-23-hardly-anyone-knows-cd.markdown]: Rephrase as "Not everyone knows what CD is". I'm just trying to mellow it down a bit :)

File: _posts/2015-06-23-hardly-anyone-knows-cd.markdown (hunk 2 above)
Comment (kmugrage) [inline _posts/2015-06-23-hardly-anyone-knows-cd.markdown]: I think your version implies that most people do, which is not true. Honestly, it's meant to be a bit abrupt.

File: _posts/2015-06-23-hardly-anyone-knows-cd.markdown (hunk 3 above)
Comment (kmugrage) [inline _posts/2015-06-23-hardly-anyone-knows-cd.markdown]: same comment as above

**Pull request merged**